src/Sylius/Bundle/CoreBundle/DependencyInjection/Configuration.php

```diff
@@ -54,6 +54,10 @@ public function getConfigTreeBuilder(): TreeBuilder
                     ->setDeprecated('sylius/sylius', '1.10', 'The "%path%.%node%" parameter is deprecated and will be removed in 2.0.')
                     ->defaultFalse()
                 ->end()
+                ->integerNode('order_token_length')
+                    ->defaultValue(64)
+                    ->min(1)->max(255)
```

Comment:
At least `10` as min value, wdyt?
```suggestion
                    ->min(10)->max(255)
```